App.js

```diff
@@ -0,0 +1,250 @@
+    
+    if(Doctor != null ){
+    if (await bcrypt.compare(req.body.Password, Doctor.doctorPassword)) 
+        {   
+            
+            
+            //access token for buyer
+            const accessToken = jwt.sign({Doctor}, process.env.DOCTOR_ACCESS_TOKEN_SECRET, {expiresIn: "60m"})
+            //refresh token for seller
+            const refreshToken = jwt.sign({Doctor}, process.env.DOCTOR_REFRESH_TOKEN_SECRET, {expiresIn: "20m"})
+            refreshTokens.push(refreshToken)
+            res.json ({accessToken: accessToken, refreshToken: refreshToken})
+        } 
+    }
+    else if(Patient != null ){
+        if (await bcrypt.compare(req.body.Password, Patient.patientPassword)) {
+            //access token for seller
+            const accessToken = jwt.sign({Patient}, process.env.PATIENT_ACCESS_TOKEN_SECRET, {expiresIn: "60m"})
+            //refresh token for buyer
+            const refreshToken = jwt.sign({Patient}, process.env.PATIENT_REFRESH_TOKEN_SECRET, {expiresIn: "20m"})
+            refreshTokens.push(refreshToken)
+            res.json ({accessToken: accessToken, refreshToken: refreshToken})
+        
+            }
+        }
+    else if (Doctor == null || Patient == null) 
+    {
+        
+        res.status(404).send ("User does not exist!")
+    }
+    else {
+    res.status(401).send("Password Incorrect!")
+    }
+   
+})
+
+
+/**
+ * @swagger
+ * /searchDoctor/{doctorName}:
+ *  get:
+ *      summary: For Searching a doctor  
+ *      description: For Searching a doctor
+ *      parameters:
+ *          - name: doctorName
+ *            in: query
+ *            description: fetch with Doctor information
+ *            schema:
+ *              type: string
+ *          - name: page
+ *            in: query
+ *            description: page number you want to go
+ *            schema:
+ *              type: number 
+ *                      
+ *      responses:
+ *          200:
+ *              description: updated Successfully
+ *              content:
+ *                  application/json:
+ *                      schema:
+ *                          type: array
+ *                          items:
+ *                              $ref: '#/components/schemas/doctorSearch'
+ *              
+ */
+
+ app.get("/searchDoctor/:doctorName",async(req,res) =>{
+    let search = req.query.doctorName
+    let page = 1
+    if(req.query.page){
+        page = req.query.page
+    }
+    let limit = 3
+    
+    let doctors = await Doctors.find({doctorName:{$regex:'.*' + search + '.*',$options:'i'}})
+    .limit(limit * 1)
+    .skip((page -1) * limit)
+    .exec();
+
+    let count = await Doctors.find({doctorName:{$regex:'.*' + search + '.*',$options:'i'}})
+    .countDocuments();
+    let totalPages = Math.ceil(count/limit)
+    let currentPage = page
+    res.json({
+        SearchResult: doctors,
+        TotalPages: totalPages,
+        currentPage: parseInt(currentPage)})
+})
+
+
+
+
```

Comment:
No routes should be defined outside of the routes folder. All the routes should be defined within the route folder.

Reply:
Added routes to its separate folder and removes uncessary lines of code from app.js[Solution](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#r1045072320)

Reply:
👍